### PR TITLE
Add a skeleton package:ok_http

### PR DESCRIPTION
**Pull request by brianquinlan**
- Most of this was generated by `flutter create --template=plugin_ffi --platforms=android ok_http`
- I updated:
  - README.md
  - CHANGELOG.md
  - pubspec.yaml
  - analysis_options.yaml
- I removed src/
- I added okhttp.yaml

---

- [X] I’ve reviewed the contributor guide and applied the relevant portions to this PR.

<details>
  <summary>Contribution guidelines:</summary><br>

- See our [contributor guide](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md) for general expectations for PRs.
- Larger or significant changes should be discussed in an issue before creating a PR.
- Contributions to our repos should follow the [Dart style guide](https://dart.dev/guides/language/effective-dart) and use `dart format`.
- Most changes should add an entry to the changelog and may need to [rev the pubspec package version](https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#making-a-change).
- Changes to packages require [corresponding tests](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md#Testing).

Note that many Dart repos have a weekly cadence for reviewing PRs - please allow for some latency before initial review feedback.
</details>

### CODE REVIEW

File: pkgs/ok_http/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  plugin_platform_interface: ^2.0.2
+
+dev_dependencies:
+  dart_flutter_team_lints: ^2.0.0
+
+flutter:
+  plugin:
+    platforms:
+      android:
+        ffiPlugin: true
```
Comment (natebosch) [inline pkgs/ok_http/pubspec.yaml:24]: [nit] Add a trailing newline.

File: pkgs/ok_http/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## 0.1.0
```
Comment (natebosch) [inline pkgs/ok_http/CHANGELOG.md:1]: This should be `-wip` until we have a release-ready package.
```suggestion
## 0.1.0-wip
```

File: pkgs/ok_http/pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+name: ok_http
+version: 0.1.0
```
Comment (natebosch) [inline pkgs/ok_http/pubspec.yaml:2]:
```suggestion
version: 0.1.0-wip
```

**Review by natebosch — APPROVED** [review]

File: pkgs/ok_http/pubspec.yaml (hunk 1 above)
Comment (brianquinlan) [reply pkgs/ok_http/pubspec.yaml]: Done

**Pull request merged**